### DISCUSSION TITLE
Info on what the different WI-FI options mean practically

### DISCUSSION THREAD

**Discussion by Jcwscience** [svpcom/wfb-ng#191] [Questions]
I've been able to get the stream working, now I'm looking to try and fine-tune it. Reading through the wiki some of the parameters are pretty self explanatory (Channel Number etc.), but others are a little more obscure for me.

I was hoping someone could describe not just what they do, but what they practically mean for real use. Like for instance 5GHz sends more data but is more easily blocked by obstacles, or something to that effect.

I know the mcs_index is the modulation type like BPSK QPSK 16-QAM, but would it be worth increasing it? And I think ldpc has to do with forward error correction, would that increase the latency or just improve link quality? Or the Reed Solomon parameters.

My setup is a little convoluted so it's tough to quickly test different options by brute force.

Thanks!

**Comment by svpcom** [comment]
Starting points:
https://en.wikipedia.org/wiki/Friis_transmission_equation
[802.11n-and-802.11ac-MCS-SNR-and-RSSI.pdf](https://github.com/svpcom/wifibroadcast/files/7837097/802.11n-and-802.11ac-MCS-SNR-and-RSSI.pdf)

TL;DR
Higher frequency - more channels, but shorter distance
Higher modulation speed - more bandwidth, but shorter distance
LDPC working for packet bits (where Reed Solomon can recover missed packet) so it should be on if it supported by your wifi adapters (works with rtl8812au, but doesn't support older ralink rt28xx)

**Reply by Jcwscience** [reply]
@svpcom Got it! I can tell I’m going to be on a Wikipedia rabbit hole with the Friis transmission equation. One more question if you have a moment, what is the port for the ssh tunnel? It’s probably written somewhere on the wiki but I haven’t been able to find it. And I assume you just run “ssh username@172.0.0.1:port”?

Thanks again!

**Comment by svpcom** [comment]
`ssh username@10.5.0.1` -- ssh to GS
`ssh username@10.5.0.2` -- ssh to Drone
see `telemetry/conf/master.cfg`

**Reply by Jcwscience** [reply]
Got it! Now we’re cooking. Now I just have to figure out what component caught my drone on fire and how to fix the damage…

**Comment by Jcwscience** [comment]
@svpcom is there anything specific I need to change in the wifibroadcast.cfg to use multiple receivers? I’ve set both interfaces in /etc/default/wifibroadcast.

I’ve got one adapter on the drone and 2 on the ground.

**Reply by svpcom** [reply]
To use TX/RX diversity you just need to add these adapters to `/etc/default/wifibroadcast` and disable them in NetworkManager config file.
